packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -362,3 +440,83 @@ describe('calculateScrollTargetForLeftSide', () => {
         ).to.equal(0);
     });
 });
+
+async function repeatScroll(
+    options: {
+        times: number;
+        elementToUpdate: TabsOverflow;
+        elementToScroll: HTMLElement;
+        distanceToReachInIteration: (iteration: number) => number;
+    },
+    iteration = 1
+): Promise<void> {
+    const {
+        times,
+        elementToUpdate,
+        elementToScroll,
+        distanceToReachInIteration,
+    } = options;
+    if (iteration > times) return;
+
+    const distanceToReach = distanceToReachInIteration(iteration);
+
+    await sendKeys({ press: 'Enter' });
+    await elementUpdated(elementToUpdate);
+    await waitUntil(
+        () =>
+            Math.ceil(Math.abs(elementToScroll.scrollLeft)) -
+                Math.abs(distanceToReach) ===
```

Comment:
Is this condition reliable enough? I'm just wondering if there’s any small discrepancy due to floating-point rounding. Alternatively we could check if it's less than a tolerance small enough, e.g. `< 0.5`. 
(but if this is not a problem, feel free to ignore!)

Reply:
I understand your concern. From my testing sessions I found the rounding up with `Math.ceil` to be enough.

Reply:
Great!